ncm-ssh/src/main/perl/ssh.pm

```diff
@@ -64,7 +77,7 @@ sub handle_config_file
         backup => '.old'
     );
 
-    foreach my $option_set (qw(comment_options options)) {
+    foreach my $option_set (qw(comment_options options multiline_options)) {
```

Comment:
yes, it will work as you describe, and as you say, it will be consistent. i guess that e.g. `HostKey` are never set by the distributed configuration file, so at least in the case of `HostKey`, it will not trigger an unnecessary restart.